docs/source/introduction/howdoesitwork.rst

```diff
@@ -18,9 +18,9 @@ OpenKAT uses a configurable data model. All information is translated into objec
 Basic concepts
 ==============
 
-Central to OpenKAT are objects and the data model. Objects are created by collecting and analyzing information. The objects found are analyzed using business rules, leading to findings that are included as objects in the data model.
+Central to OpenKAT are objects and `the data model, which defines objecttypes <https://mispo.es/model-explorer/model-explorer.html>`_. Objects (or 'objects of interest, ooi') are created by collecting and analyzing information. The objects found are analyzed using business rules, leading to findings that are included as objects in the data model.
```

Comment:
https://mispo.es has an expired certificate. Can't we make this page part of the documentation instead?

Reply:
Perfect idea, will discuss today with @Darwinkel

Reply:
I agree with @praseodym that we shouldn't link to things that have expired certs. Other than that, the model explorer page is also outdated and misses newer objects.

Reply:
Mispo.es is part of our 'test bed' and should not be hosting anything we link to in the docs.
If anywhere, I'd like the model explorer to be part of the OpenKAT install itself, reading from the Models available to Octopoes (as those could become plugins themselves at some point), and it helps developers who work on the model to see if their changes are loaded.

Reply:
I have removed the link to mispo.es.